### DISCUSSION TITLE
Easy Installation of Coder

### DISCUSSION THREAD

**Discussion by nooobcoder** [coder/code-server#2759] [Ideas] [answered]
Codercom is providing 60 days free trial to it's coder platform. I am talking specifically about coder and not code-server. I got not help of a docker installation. All docs were through Kubernetes. I strictly need a docker guide to get easy with the deployment. Someone reading this, please guide me a docker compose version of deploying a light version of coder-enterprise. 

Thank You

**Comment by bpmct** [comment, marked as answer]
@nooobcoder We currently don't have a docker-compose version of Coder, but we are currently considering options like this.

For now, you can install Coder with kind, which runs a small Kubernetes cluster inside of Docker. There are not a lot of requirements and it just takes one command to set up. [https://coder.com/docs/setup/kubernetes/local-preview](https://coder.com/docs/setup/kubernetes/local-preview)


If you need help with this or have more questions, please reach out on [our Slack channel](https://cdr.co/join-community). I am very active on there and willing to help with any questions :)

**Reply by sr229** [reply]
Additionally, you can also run the quickstart in Docker Desktop via the Kubernetes Engine. There are no plans for Coder to support Docker itself by all means since it relies on a  lot of Kubernetes resource definitions.
